torch/fx/experimental/symbolic_shapes.py

```diff
@@ -765,15 +765,15 @@ def evaluate_guards_for_args(self, placeholders, args):
         return eval(code, {}, dict(zip(arg_names, args)))
 
     def get_nontrivial_guards(self):
-        return [self.simplify(guard) for guard, _ in self.guards if self._maybe_evaluate_static(guard) is None]
+        return [self.simplify(guard.expr) for guard in self.guards if self._maybe_evaluate_static(guard.expr) is None]
```

Comment:
Why are these changing?

Reply:
Readability

Reply:
to elaborate: I could rename this from guard to something like:

`return [self.simplify(expr) for expr, _ in self.guards if self._maybe_evaluate_static(expr) is None]`

But that felt less readable than this